src/components/MaplibreMap.jsx

```diff
@@ -201,15 +201,13 @@ const MaplibreMap = ({ view, setView }) => {
     const map = mapRef.current.getMap()
     const { lat, lng } = map.getCenter()
     const zoom = map.getZoom()
-    const queryParams = getURLParams()
+    const newQueryParams = getURLParamsSnapshot()
     const bounds = map.getBounds()
 
-    queryParams.set('lat', lat)
-    queryParams.set('lng', lng)
-    queryParams.set('zoom', zoom)
-    updateURLParams(queryParams)
-    setMapCenter([lat, lng])
-    setMapZoom(zoom)
```

Comment:
We only used map center and map zoom for initial zoom settings. No need to update it on map move.